### ISSUE TITLE
Evaluate Traffic Light Classifier

### ISSUE THREAD

**Issue by salaniz** [salaniz/CarND-DriveIT-System-Integration#2]
We need some statistics to measure how good our classifier is currently.

1. Build a test set from simulator images and download the test images from the real car (see classroom).
2. Run the classifier and report scores.

**Comment by shangliy** [comment]
I will do that. One question confusing me is do we need two model to handle either simulator data or real data. Considering the images quite differ from each other.

**Comment by salaniz** [comment]
We will have to try that out, I believe. One would be better because we could use more data for the model, but if the simulator data is too artificial that it reduces the accuracy for real traffic lights then we should do separate ones.

**Comment by shangliy** [comment]
I have retrain the models for both sim and real images. The models are under the
 " CarND-DriveIT-System-Integration/tools/car_detector/models/ "
I also added the test script "model_test.py" to test models and do inference. You can modify the test_img_dir and model as well as output dir. 
The current inference evaluation results are under /tools/car_detector/result/real/ , tools/car_detector/result/sim/ and /tools/car_detector/result_camera_capture/.

**Comment by salaniz** [comment]
Thanks a lot, I think the results look good and we can leave it like this.

**Comment by salaniz** [comment]
I am reopening this issue, because according to @zhangjinge588 the classification on the real images can be improved and also I noticed that traffic lights are not detected towards the end of the highway route when you go the full loop.

Could it be that training data is a bit skewed towards images from the first traffic lights? There is works very well.

Which script did you use to collect the data? I could help with that.
